Dockerfile

```diff
@@ -26,6 +26,12 @@ ENV SPARK_HOME=/opt/spark
 # Add scripts used to configure the image
 COPY scripts /tmp/scripts
 
+# Adding jmx by default
+COPY metrics /opt/spark
+RUN mv $SPARK_HOME/metrics.properties $SPARK_HOME/conf/metrics.properties && chown 185 /opt/spark/conf/metrics.properties && chmod a+rwx /opt/spark/jolokia-jvm-1.3.6-agent.jar && chown 185 /opt/spark/jolokia-jvm-1.3.6-agent.jar
+
```

Comment:
Instead of doing this with extra COPY/RUN commands, the metrics.properties file should be added to scripts/spark/added and then moved in scripts/spark/install like the spark-defaults.conf and log4j.properties files that already exist.